### PR TITLE
Colors: Update color library

### PR DESCRIPTION
**Pull request by franciscrispin**
https://github.com/glints-dev/glints-aries/issues/148

- Remove some colors from SecondaryColor
- Add colors to Grayscale

Glints UI: https://app.zeplin.io/project/5ba343bf0c399c33db483653/screen/5bbf16daf616cc09a6264519

### CODE REVIEW

**Comment by franciscrispin** [comment]
@aaa59891 PTAL :pray: I decided not to show the deprecation messages because 

1. I can just refactor all the instances of the deprecated colors in our codebases
2. When I showed the deprecation message using the `IIFE`, the moment you import the color, all the deprecation messages will show at the same time which is not very helpful

File: src/Style/Colors.ts [hunk 1]
```diff
@@ -1,31 +1,47 @@
...
   lightgreen: '#EEF6F4',
-  darkgreen: '#008464',
-  orange: '#F5A623',
   lightorange: '#FFF6E8',
   lightred: '#FFF2F3',
   lightblue: '#D6F4FF',
   darkblue: '#004b68',
+
+  // renamed
+  purplepizza: '#FF00E5',
```
Comment (aaa59891) [inline src/Style/Colors.ts]: oh there two are renamed, but it seems they are duplicated, and in the storybook, I only see `magenta` and `violet`, which two do you want to be used?

Comment (franciscrispin) [reply src/Style/Colors.ts]: We will use this:
```
magenta: '#FF00E5',
violet: '#9013FE',
```

I'm leaving `purplepizza` and `bluegem` there until I refactor all the usage of them in our codebases.

Comment (aaa59891) [reply src/Style/Colors.ts]: ok!

**Comment by aaa59891** [comment]
LGTM

**Pull request merged**